tools/scripts/generic.mk

```diff
@@ -8,6 +8,10 @@ rwildcard = $(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2
 #                          COMMON INITIALIZATION
 #------------------------------------------------------------------------------
 
+#This compiler flags will be added to project if they are found in CFLAGS
+ADDITONAL_FLAGS = XILINX_PLATFORM ADUCM_PLATFORM ALTERA_PLATFORM \
```

Comment:
I'm not sure if this is the right approach. If I want to add a completely new flag, I need to edit this line too which is not ideal.

We should propagate everything that starts with -D from CFLAGS to the SDK.

Reply:
done